### ISSUE TITLE
displayFormat and SaveFormat don't work in last upgrade

### ISSUE THREAD

**Issue by paskuale75** [kartik-v/yii2-datecontrol#107]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest `master` branch of yii2-datecontrol.
- [x] This is not an usage question. I confirm having gone through and read the [documentation](http://demos.krajee.com/datecontrol) and [demo](http://demos.krajee.com/datecontrol-demo).
- [x] This is not a general programming / coding question. (Those should be directed to the [webtips Q & A forum](http://webtips.krajee.com/questions)).
- [x] I have attempted to find the simplest possible steps to reproduce the issue.
- [x] I have included a failing test as a pull request (Optional).

## Steps to reproduce the issue

1. my config/main.php :

```
'datecontrol' =>  [
            'class' => '\kartik\datecontrol\Module',

            // format settings for displaying each date attribute (ICU format example)
            'displaySettings' => [
                Module::FORMAT_DATE => 'php:d/m/Y',
                Module::FORMAT_TIME => 'HH:mm:ss a',
                Module::FORMAT_DATETIME => 'dd-MM-yyyy HH:mm:ss a',
            ],

            // format settings for saving each date attribute (PHP format example)
            'saveSettings' => [
                Module::FORMAT_DATE => 'php:Y-m-d', // saves as unix timestamp
                Module::FORMAT_TIME => 'php:H:i:s',
                Module::FORMAT_DATETIME => 'php:Y-m-d H:i:s',
            ],

            // set your display timezone
            'displayTimezone' => 'Europe/Paris',

            // set your timezone for date saved to db
            'saveTimezone' => 'Europe/Paris',

            // automatically use kartik\widgets for each of the above formats
            'autoWidget' => true,

            // use ajax conversion for processing dates from display format to save format.
            'ajaxConversion' => true,

            // default settings for each widget from kartik\widgets used when autoWidget is true
            'autoWidgetSettings' => [
                Module::FORMAT_DATE => ['type'=>2, 'pluginOptions'=>['autoclose'=>true]], // example
                Module::FORMAT_DATETIME => [], // setup if needed
                Module::FORMAT_TIME => [], // setup if needed
            ],

            // custom widget settings that will be used to render the date input instead of kartik\widgets,
            // this will be used when autoWidget is set to false at module or widget level.
            'widgetSettings' => [
                Module::FORMAT_DATE => [
                    'class' => 'yii\jui\DatePicker', // example
                    'language' => 'it',
                    'options' => [
                        'dateFormat' => 'php:Y-m-d',
                        'options' => ['class'=>'form-control'],
                    ]
                ]
            ]
            // other settings
        ],...
```

2. in the View (form.php)
  ```
$form->field($model->anagraficanascita, 'data',['options' => ['class' => 'col-md-4']])
        ->widget(DateControl::classname());
```

3.

## Expected behavior and actual behavior

When I follow those steps, I see dateFormat **Jan 01, 2017** 

I was expecting... **01/01/2017**

#### Browsers

- [ ] Google Chrome
- [X ] Mozilla Firefox
- [ ] Internet Explorer
- [ ] Safari

#### Operating System

- [ ] Windows
- [ ] Mac OS X
- [X ] Linux
- [ ] Mobile

#### Libraries

- jQuery version:
- yii2-datecontrol version: last

## Isolating the problem

- [ ] This bug happens [on the demos page](https://demos.krajee.com/datecontrol-demo)
- [ ] The bug happens consistently across all tested browsers
- [ ] This bug happens when using yii2-datecontrol without other plugins.

**Comment by kartik-v** [comment]
You would need to debug and check your module configuration and widget settings. 

To understand the configuration go to the [date control demo page](http://demos.krajee.com/datecontrol-demo) and play around with various values - and let me know if you can reproduce your issue.

**Comment by kartik-v** [comment]
Will be resolved via #108.
